app/config/locale/translations/pt-pt.json

```diff
@@ -3,30 +3,30 @@
     "settings.locale": "pt-pt",
     "settings.direction": "ltr",
     "emails.sender": "Equipa %s",
-    "emails.verification.subject": "",
-    "emails.verification.hello": "",
-    "emails.verification.body": "",
-    "emails.verification.footer": "",
-    "emails.verification.thanks": "",
-    "emails.verification.signature": "",
-    "emails.magicSession.subject": "",
-    "emails.magicSession.hello": "",
-    "emails.magicSession.body": "",
-    "emails.magicSession.footer": "",
-    "emails.magicSession.thanks": "",
-    "emails.magicSession.signature": "",
-    "emails.recovery.subject": "",
-    "emails.recovery.hello": "",
-    "emails.recovery.body": "",
-    "emails.recovery.footer": "",
-    "emails.recovery.thanks": "",
-    "emails.recovery.signature": "",
-    "emails.invitation.subject": "",
-    "emails.invitation.hello": "",
-    "emails.invitation.body": "",
-    "emails.invitation.footer": "",
-    "emails.invitation.thanks": "",
-    "emails.invitation.signature": "",
+    "emails.verification.subject": "Verificação de contas",
+    "emails.verification.hello": "Hey {{name}}",
+    "emails.verification.body": "Siga esta ligação para verificar o seu endereço de correio electrónico.",
+    "emails.verification.footer": "Se não pediu para verificar este endereço, pode ignorar esta mensagem.",
+    "emails.verification.thanks": "Obrigado",
+    "emails.verification.signature": "Equipa {{project}}",
+    "emails.magicSession.subject": "Login",
+    "emails.magicSession.hello": "Olá ,",
+    "emails.magicSession.body": "Siga esta ligação para iniciar sessão.",
+    "emails.magicSession.footer": "Se não pediu para entrar usando este e-mail, pode ignorar esta mensagem.",
+    "emails.magicSession.thanks": "Obrigado",
+    "emails.magicSession.signature": "Equipa {{project}}",
+    "emails.recovery.subject": "Redefinição de senha",
+    "emails.recovery.hello": u"Olá {{name}}",
+    "emails.recovery.body": "tilize este link para redefinir a palavra-passe do seu projecto {{project}}",
+    "emails.recovery.footer": "Se não pediu para redefinir a sua palavra-passe, pode ignorar esta mensagem.",
+    "emails.recovery.thanks": "Obrigado",
+    "emails.recovery.signature": "Equipa {{project}}",
+    "emails.invitation.subject": "Convite à equipa de %s às %s",
+    "emails.invitation.hello": "Olá",
+    "emails.invitation.body": "Este correio foi-lhe enviado porque {{{owner}} queria convidá-lo a tornar-se membro da equipa {{{team}} da {{project}}.",
```

Comment:
```suggestion
    "emails.invitation.body": "Este correio foi-lhe enviado porque {{owner}} queria convidá-lo a tornar-se membro da equipa {{team}} da {{project}}.",
```

Reply:
@christyjacob4 thanks for the feedback. The file is already updated :)